isso/js/app/i18n.js

```diff
@@ -2,8 +2,8 @@ define(["app/config", "app/i18n/bg", "app/i18n/cs", "app/i18n/da",
         "app/i18n/de", "app/i18n/en", "app/i18n/fa", "app/i18n/fi",
         "app/i18n/fr", "app/i18n/hr",  "app/i18n/hu", "app/i18n/ru", "app/i18n/it",
         "app/i18n/eo", "app/i18n/pl", "app/i18n/sv", "app/i18n/nl", "app/i18n/el_GR",
-        "app/i18n/es", "app/i18n/vi", "app/i18n/zh_CN", "app/i18n/zh_CN", "app/i18n/zh_TW"],
-        function(config, bg, cs, da, de, en, fa, fi, fr, hr, hu, ru, it, eo, pl, sv, nl, el, es, vi, zh, zh_CN, zh_TW) {
+        "app/i18n/es", "app/i18n/vi", "app/i18n/zh_CN", "app/i18n/zh_CN", "app/i18n/zh_TW", "app/i18n/sk"],
```

Comment:
Please keep this alphabetical.

Reply:
I corrected my changes, although it is still not alphabetically correct. Should I fix all imports?

Reply:
That's okay, at least this is not making it worse.